docs/css_types/_template.md

```diff
@@ -0,0 +1,40 @@
+<!-- Template file for a Textual CSS type reference page. -->
+
+# &lt;type-name&gt;
+
+<!-- Short description of the type. -->
+
+## Syntax
+
+<!--
+--8<-- "docs/snippets/type_syntax/type_name.md"
+-->
+
+## Examples
+
+### CSS
+
+<!--
+Examples should be rule-agnostic.
+Include a good variety of examples.
+If the type has many different syntaxes, cover all of them.
+Add comments when needed/if helpful.
+-->
+
+```sass
+* {
+    rule: type-value-1;
+    rule: type-value-2;
+    rule: type-value-3;
+}
+```
+
+### Python
+
+<!-- Same examples as above. -->
+
+```py
+type_name = type_value_1
```

Comment:
I think we need to need to show that these are properties on Widget somehow. 

```python
my_widget.rule = value
```

Reply:
Fixed in 81ddae7